tests/features/content_editing_admin.feature

```diff
@@ -6,6 +6,11 @@ Feature: Content editing as administrator
 
   Background:
     Given I am logged in as a user with the 'administrator' role
+    And the following languages are available:
```

Comment:
This 'Given' step should be only in the scenario. 
Because here, it adds useless step for the other scenarios.